src/fcBindingModel.jl

```diff
@@ -9,9 +9,18 @@ function polyfc_ActV(L0, KxStar, f, Rtot::Array, IgGC::Array, Kav::AbstractMatri
     """
     ansType = promote_type(typeof(L0), typeof(KxStar), typeof(f), eltype(Rtot), eltype(IgGC))
     res = Matrix{ansType}(undef, size(Rtot, 2), size(IgGC, 2))
-    for ict = 1:size(res, 1)
-        for iset = 1:size(res, 2)
-            res[ict, iset] = polyfc(L0, KxStar, f, Rtot[:, ict], IgGC[:, iset], Kav, ActI).ActV
+    if Mix
+        for ict = 1:size(res, 1)
+            for iset = 1:size(res, 2)
+                res[ict, iset] = polyfc(L0, KxStar, f, Rtot[:, ict], IgGC[:, iset], Kav, ActI).ActV
+            end
+        end
+    else
+        L = (range(0, stop = 1, length = size(res, 2)))*L0
```

Comment:
If you assign this to `L0` if Mix is false, then you can move the loops out of the if statement. That's probably cleaner.